openmetrics/src/test/resources/expected-metrics.txt

```diff
@@ -0,0 +1,513 @@
+# TYPE io_airlift_http_client_WrittenBytes_AllTime_Total gauge
```

Comment:
Metrics for both `ForFirstService` and `ForSecondService` should be grouped under a single `# TYPE`, but this is not implemented yet. I wanted to get an initial approval of this approach first.

Reply:
In the openmetrics spec they should be, they should also share the same help section.